pkg/volume/csi/csi_attacher_test.go

```diff
@@ -474,7 +474,7 @@ func TestAttacherMountDevice(t *testing.T) {
 			devicePath:      "",
 			deviceMountPath: "path2",
 			stageUnstageSet: true,
-			shouldFail:      true,
+			shouldFail:      false,
```

Comment:
Currently `csiAttacher.MountDevice` doesn't use the input parameter `devicePath`, so the it will succeed even if `devicePath` missed.

Reply:
why doesn't this test case pass for both true and false without change to the test logic? Something must not be working properly. I agree this change is correct but can you fix the test so that the test will fail when `shouldFail` is set to `true`.

Reply:
Actually this test case can't pass for `shouldFail = true`, but due to existence of the [first bug](https://github.com/kubernetes/kubernetes/blob/28f9356ad1183f8ede3a1c92a0e92ac137fab6c4/pkg/volume/csi/csi_attacher_test.go#L542), the test function is returned too early, so this test case has never been executed.
I encountered this error when I try to fix the first bug, so I fix it in this PR too.

Reply:
I see, LGTM